### PR TITLE
Add draft 2020 AGM minutes

### PR DESCRIPTION
**Pull request by dcwalk**
Draft minutes for the AGM for approval

### CODE REVIEW

File: _posts/members-meetings/2020-06-13-agm-minutes.md [hunk 1]
```diff
@@ -0,0 +1,836 @@
+# Hypha 2020 AGM - June 13-14, 2020
+
+_These are draft minutes from our initial AGM._
```
Comment (benhylau) [inline _posts/members-meetings/2020-06-13-agm-minutes.md:3]:
```suggestion
_These are minutes from our inaugural AGM._
```

File: _posts/members-meetings/2020-06-13-agm-minutes.md [hunk 2]
```diff
@@ -0,0 +1,836 @@
...
+  - **Finance (Finance Statements)**
+    - accomplishments
+        - new $$$ accounts: bank, transferwise, opencollective (future incoming flows), paypal (incoming), ...
+        - new CRA accounts: 3 types: income tax, payroll, and HST.
+        - payroll process setups
+    - payments
+        - pay wages+expenses
+        - pay contractors
+        - invoice clients and collect
+        - can receive: CAD, USD, EU, UK
```
Comment (benhylau) [inline _posts/members-meetings/2020-06-13-agm-minutes.md:158]:
```suggestion
        - can receive: CAD, USD, EUR, GBP
```

File: _posts/members-meetings/2020-06-13-agm-minutes.md [hunk 3]
```diff
@@ -0,0 +1,836 @@
...
+|------|--------|
+| dc   | 2      |
+| pc   | 3      |
+| el   | 3      |
+| bl   | 5      |
+| yj   | 2      |
+| uv   | 2      |
+| gi   | 5      |
+
+## Appendix - Annual Resolution of the Board
```
Comment (benhylau) [inline _posts/members-meetings/2020-06-13-agm-minutes.md:643]: This needs to move into a resolution and merged with existing template.

File: _posts/members-meetings/2020-06-13-agm-minutes.md (hunk 1 above)
Comment (dcwalk) [reply _posts/members-meetings/2020-06-13-agm-minutes.md:3]: If they haven't been approved we need to indicate that somewhere

File: _posts/members-meetings/2020-06-13-agm-minutes.md [hunk 4]
```diff
@@ -0,0 +1,836 @@
...
+
+2.  The directors of the board unanimously adopt an amendment of the articles for adoption 
+
+3.  [  ] of the Co-operative is authorized to sign all documents and perform such acts as may be necessary or desirable to give effect to the above resolution.
+
+4.  The following individuals are appointed and confirmed as officers of the Co-operative for a term of one year or until replaced:
+
+Yurko J: President; 
+Patrick Connolly: Vice-President; 
+Udit Vira: Teasurer; and
```
Comment (benhylau) [inline _posts/members-meetings/2020-06-13-agm-minutes.md:669]:
```suggestion
Udit Vira: Treasurer; and
```

**Comment by benhylau** [comment]
Merging these, pls fix remaining issues in subsequent PR.

**Pull request merged**